microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,528 @@
+
+**Unity**, a versatile game engine, is a popular choice for creating 3D applications, including mixed reality experiences for HoloLens 2 devices, due to its extensive features and robust capabilities for AR/VR/MR.
+
+The **Mixed Reality Toolkit (MRTK)** is a Microsoft-driven open source project designed to accelerate the development of mixed reality applications in Unity. MRTK provides a collection of scripts, components, and prefabs specifically designed for building performant and intuitive mixed reality experiences on HoloLens 2 devices.
+
+In Unity, the **Microsoft Mixed Reality WebView** plugin can be installed via the **MRTK Feature Tool**. The WebView plugin for Unity simplifies the integration of WebView2 functionality into your HoloLens 2 app by wrapping the WebView2 control and automatically handling rendering and directing input to the WebView2 control. The plugin also manages interop between Unity and WebView2, enabling communication between JavaScript and Unity via messages and events.
+
+By combining HoloLens 2, Unity, MRTK, and WebView2, you can create immersive mixed reality experiences that seamlessly integrate web content.
+
+<!-- ====================================================================== -->
+## Step 1 - Install Unity and Visual Studio
+
+This tutorial assumes you have Unity 2020.3 LTS or later and Visual Studio 2019 version 16.9 or later installed. Visual Studio 2017 isn't supported.
+
+Follow the [Mixed Reality tools installation guide](../../../windows/mixed-reality/develop/install-the-tools?tabs=unity) to set up your Unity and Visual Studio environment for HoloLens 2 development. These resources will guide you through the process of installing and configuring the necessary tools, including Visual Studio and the Universal Windows Platform development workload.
+
+Additionally, ensure that your development environment is set up for WebView2 by following the instructions on [Set up your Dev environment for WebView2](../how-to/machine-setup).
+
+Once you have completed the setup, return to this page and continue with the next steps.
+
+<!-- ====================================================================== -->
+## Step 2 - Install MRTK and MRTK Feature Tool
+
+1. To install the Mixed Reality Toolkit (MRTK) and the MRTK Feature Tool, first [Download the Mixed Reality Feature Tool](../../../windows/mixed-reality/develop/unity/welcome-to-mr-feature-tool) from the Microsoft Download Center. The MRTK Feature Tool simplifies the process of discovering, installing, and managing Mixed Reality features and packages for your Unity project. For more guidance on installing the Mixed Reality Feature Tool, refer to [this page](../../../windows/mixed-reality/develop/unity/welcome-to-mr-feature-tool) for more information.
```

Comment:
```suggestion
1. To install the Mixed Reality Toolkit (MRTK) and the MRTK Feature Tool, first [Download the Mixed Reality Feature Tool](/windows/mixed-reality/develop/unity/welcome-to-mr-feature-tool) from the Microsoft Download Center. The MRTK Feature Tool simplifies the process of discovering, installing, and managing Mixed Reality features and packages for your Unity project. For more guidance on installing the Mixed Reality Feature Tool, refer to [this page](../../../windows/mixed-reality/develop/unity/welcome-to-mr-feature-tool) for more information.
```
different doc-set/repo, so start w/ slash, end w/ no .md

https://learn.microsoft.com/windows/mixed-reality/develop/unity/welcome-to-mr-feature-tool

Reply:
@michaelfarnsworth -- check the URL i changed to -- https://www.microsoft.com/download/details.aspx?id=102778 -- and then close this thread